reqwest-tracing/src/reqwest_otel_span_macro.rs

```diff
@@ -1,5 +1,5 @@
 #[macro_export]
-/// [`reqwest_otel_span!`] creates a new [`tracing::Span`].
+/// [`crate::reqwest_otel_span!`] creates a new [`tracing::Span`].
```

Comment:
```suggestion
/// [`reqwest_otel_span!`](crate::reqwest_otel_span) creates a new [`tracing::Span`].
```
I think is more preferable here